src/modules/Elsa.Studio.Workflows/Components/WorkflowDefinitionEditor/Components/ActivityProperties/Tabs/VersionTab.razor

```diff
@@ -29,7 +29,7 @@
             <MudTd>@context.Version</MudTd>
             <MudTd>@context.CreatedAt.ToString("g")</MudTd>
             <MudTd style="text-align: right;">
-                <MudMenu Icon="@Icons.Material.Filled.MoreVert" Disabled="IsReadOnly">
+                <MudMenu Icon="@Icons.Material.Filled.MoreVert" Disabled="IsReadOnly || !_versionsUsableAsActivity.Contains(context.Version)" Title="@(!_versionsUsableAsActivity.Contains(context.Version) ? "Not usable as activity." : "")">
```

Comment:
Will this not give an empty pop-up for versions that are usable as activity? I think it would be better to encapsulate the entire Title in an if-statement

Reply:
There won't be an empty pop-up, but code-wise, might be better to encapsulate the whole title.

Reply:
Apparently, Blazor does not support that.